frontend/talentsearch/src/js/components/diversityEquityInclusion/DiversityEquityInclusion.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import type { Meta, Story } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { fakeUsers } from "@common/fakeData";
+import { User } from "../../api/generated";
```

Comment:
Unused import?

Reply:
Man, I swear every PR I miss this. I need better glasses 👓 

Resolved with [remove unused imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2989/commits/19878f79437c9534a70d71917491db4691d7b662)